src/Microsoft.Health.Fhir.Shared.Core/Features/Search/SearchOptionsFactory.cs

```diff
@@ -90,6 +90,20 @@ public SearchOptions Create(string compartmentType, string compartmentId, string
                     // Query parameter with empty value is not supported.
                     unsupportedSearchParameters.Add(query);
                 }
+                else if (string.Compare(query.Item1, KnownQueryParameterNames.Total, StringComparison.OrdinalIgnoreCase) == 0)
+                {
+                    if (Enum.TryParse<TotalType>(query.Item2, true, out var totalType))
+                    {
+                        if (totalType == TotalType.Accurate)
+                        {
+                            searchOptions.IncludeTotal = TotalType.Accurate;
+                        }
+                        else if (totalType == TotalType.None)
+                        {
+                            searchOptions.IncludeTotal = TotalType.None;
+                        }
```

Comment:
Is there any reason we can't just do `searchOptions.IncludTotal = totalType;` here?

Reply:
Nope, good idea. Fixed in 85c7130.